PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
+{Please ensure that you have Javadocs written for all new methods and classes in your PR. Also, please test your PR at least once before submitting.}
```

Comment:
I don't think this is required for the PR template as long as the contributing guidelines have these instructions.

Reply:
Hmm, good point. However, does anyone actually read contributing.md before submitting their PR? :)

Reply:
Hmm, valid point. In that case we could add a link to the contributing guidelines in the PR template. 

However, PR templates is meant for just letting the contributor know what all things should be mentioned in the description for a good PR.

Reply:
True. Will modify it.